src/main/java/executor/service/service/Item.java

```diff
@@ -0,0 +1,15 @@
+package executor.service.service;
+
+/**
+ * A class implements the {@code Item} interface to
+ * indicate to the {@link executor.service.model.Scenario}
+ * and {@link executor.service.model.ProxyConfigHolder}
+ * classes that are allowed to implement this interface.
+ * <p>
+ * @see     executor.service.model.Scenario
+ * @see     executor.service.model.ProxyConfigHolder
+ * @author Oleksandr Tuleninov
+ * @version 01
+ */
+public interface Item {
```

Comment:
What semantic meaning of *Item*? How it combines the Scenario and Proxy entity?

Reply:
This point has been changed.